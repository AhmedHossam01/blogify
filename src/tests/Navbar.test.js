import React from "react";
import { render, unmountComponentAtNode } from "react-dom";
import Navbar from "../components/layout/Navbar";

describe("Navbar Component", () => {
  const container = document.createElement("div");

  it("should match the snapshot", () => {
    render(<Navbar></Navbar>, container);
    expect(container.innerHTML).toMatchSnapshot();
  });
});
