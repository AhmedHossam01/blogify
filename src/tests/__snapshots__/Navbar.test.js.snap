// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without error 1`] = `"<nav class=\\"navbar navbar-expand navbar-dark bg-dark\\"><div class=\\"container\\"><a href=\\"#home\\" class=\\"navbar-brand\\">Blogify</a><div class=\\"mr-auto navbar-nav\\"><a href=\\"#home\\" data-rb-event-key=\\"#home\\" class=\\"nav-link\\">Home</a></div><form class=\\"form-inline\\"><input placeholder=\\"Search\\" type=\\"text\\" class=\\"mr-sm-2 form-control\\"><button type=\\"button\\" class=\\"btn btn-outline-light\\">Search</button></form></div></nav>"`;
